tests/python/test_multidevice_overlap.py

```diff
@@ -88,3 +92,21 @@ def test_overlap_allgather_matmul_stream_outermost(multidevice_test, benchmark):
 
     # benchmark
     benchmark.pedantic(lambda: fd.execute(ins), rounds=N_ITERATIONS)
+
+
+@pytest.mark.mpi
+def test_overlap_allgather_matmul_stream_outermost_ucc(multidevice_test, benchmark):
```

Comment:
This sounds like a perfect place to use [pytest.mark.parametrize](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html)

Reply:
good suggestion, thanks! Done